docs/Upgrade.rst

```diff
@@ -531,24 +273,18 @@ impacted by any previous upgrade procedure.
 4. Configure the **EXO\_DATA\_DIR** variable. Refer to :ref:`Data directory configuration <Configuration.DataDirectory>`
    for more details.
 
-5. Go to the new eXo Platform package (``$PLATFORM_TOMCAT_HOME/gatein/conf/`` 
-   or ``$PLATFORM_JBOSS_HOME/standalone/configuration/gatein/``), then 
-   rename the sample upgrade file regarding the version you want to 
-   upgrade known as ``upgrade-sample.properties``) into 
-   ``upgrade.properties`` as described in `Release Notes <https://community.exoplatform.com/portal/g/:spaces:platform_4/exo_platform/wiki/group/spaces/platform_4/Release_Notes_-_Platform_5.0>`__.
-
-6. Start the eXo Platform server. The upgrade will be run automatically. 
+5. Start the eXo Platform server. The upgrade will be run automatically. 
    The startup is successful when you see a message like **INFO \| Server startup in XXXX ms**.
 
-7. Stop the server.
+6. Stop the server.
 
-8. Remove or rename the ``upgrade.properties`` in Step 5. This is to 
+7. Remove or rename the ``upgrade.properties`` in Step 5. This is to 
    avoid running the upgrade again for next time.
 
-9. Restart the server, then do some tests on the upgraded version. See
+8. Restart the server, then do some tests on the upgraded version. See
    :ref:`Best practices <Upgrade.BestPractices>` for more details.
 
-.. note::-  eXo Platform 5.1 version requires the version 5.6 of
+.. note::-  eXo Platform 5.1 version requires the version 5.6.9 of
```

Comment:
No need to precise the maintenance version, just keep 5.6.

Reply:
OK